opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/resources/mockedsandbox/restrecord/redirect/accounts/sandbox/mappings/mapping-v1-consents-1243243.json

```diff
@@ -11,7 +11,7 @@
         "matches" : "[0-9a-zA-z\\-]+"
       },
       "PSU-ID" : {
-        "equalTo" : "anton.brueckner"
+        "equalTo" : "[a-z\\.]*"
```

Comment:
The expectation should be clear if you want to use no PSU ID - you need to create a new fixture package